src/opera/util/run_utils.py

```diff
@@ -93,6 +93,60 @@ def create_sas_command_line(sas_program_path, sas_runconfig_path,
     return command_line
 
 
+def create_qa_command_line(qa_program_path, qa_program_options=None):
+    """
+    Forms the appropriate command line for executing a Quality Assurance (QA)
+    application from parameters obtained from the RunConfig.
```

Comment:
Everything looks great so far:  this is very small, but it might be desirable to have SAS called out explicitly to reinforce where the executable is coming from.  'for executing a SAS Quality Assurance ..."  May not be necessary since it's basically default behavior for this type of executable, but thought I'd throw it out there.